### PR TITLE
Add links to manage self-deposit to the sidebar

### PR DESCRIPTION
**Pull request by little9**

### CODE REVIEW

File: spec/features/manage_deposit_types_link_spec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+include Warden::Test::Helpers
+
+RSpec.feature 'Manage Deposit Types link in dashboard sidebar', :clean do
+  context 'a logged in user' do
+    let(:user_attributes) do
+      { email: 'test@example.com' }
+    end
+    let(:user) do
+      User.new(user_attributes) { |u| u.save(validate: false) }
```
Comment (no-reply) [inline spec/features/manage_deposit_types_link_spec.rb]: Do we want to use the user factories?

File: spec/features/manage_deposit_types_link_spec.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    before do
+      AdminSet.find_or_create_default_admin_set_id
+      login_as user
+    end
+
+    scenario do
+      visit '/dashboard'
+      within '.sidebar' do
+        expect(page).to have_content "Manage self-deposit types"
```
Comment (no-reply) [inline spec/features/manage_deposit_types_link_spec.rb]: Should these tests click the link and test the destination? I have the vague feeling that the tests as is are trivial enough to not be worth the setup costs.

Comment (little9) [inline spec/features/manage_deposit_types_link_spec.rb]: This is good feedback! Do you think that this kind of change, (adding links to sidebars) warrants feature tests?

Comment (no-reply) [inline spec/features/manage_deposit_types_link_spec.rb]: Yeah, let's go ahead and move this forward without the tests. I think they are just a bit fragile (changing the labels would be breaking) and aren't really testing anything that could be accidentally removed.

Maybe a test to touch the sidebar as a whole would be appropriate, but short of coming up with a more comprehensive strategy this looks about as good as we can do.

**Pull request merged**